docs/guides/configuration.rst

```diff
@@ -33,6 +33,8 @@ Here is a list of all available options
 +-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
 | adminPassword   | -                        | ``--adminPassword``      | ``hoodie_adminPassword`` | Password to login to Admin Dashboard. Login is not possible unless set                                              |
 +-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
+| name            | hoodie                   | ``--name``               | ``hoodie_name``          | Name your application.                                                                                              |
++-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
```

Comment:
the default value is not `hoodie`, but the value of the `"name"` property in the `package.json` file

Reply:
:boom: thanks man - mod going up shortly!